packages/two_dimensional_scrollables/lib/src/table_view/table_span.dart

```diff
@@ -364,15 +371,20 @@ class TableSpanDecoration {
   /// cells.
   void paint(TableSpanDecorationPaintDetails details) {
     if (color != null) {
-      details.canvas.drawRect(
-        details.rect,
-        Paint()
-          ..color = color!
-          ..isAntiAlias = false,
-      );
+      final Paint paint = Paint()
+        ..color = color!
+        ..isAntiAlias = false;
```

Comment:
For rounded corners, don't we need AA to make them look pretty? (we probably also want this if the rect is not axis aligned)

Reply:
I believe you can see the rough edges in the screenshot of the test?

Reply:
Oh yeah, let me add that conditionally